src/Network/Network/AzureFirewallPolicy/ApplicationRuleCondition/NewAzureFirewallPolicyApplicationRuleCommand.cs

```diff
@@ -70,6 +70,12 @@ public override void Execute()
         {
             base.Execute();
 
+            // One of SourceAddress or SourceIpGroup must be present
```

Comment:
if one of these two parameter is required, it need to be handled by parameter sets:

one with SourceAddress  mandatory and the other with SourceIpGroup  mandatory.